README.md

```diff
@@ -377,9 +272,13 @@ client.query(predicateExpr) { [weak self] result in
 
 ### Example project
 
-`FaunaDB.workspace` contains an Example app that shows the basics of how to use the driver and populates an iOS UITableView with fauna db data as well as paginate and filter fauna db data. It shows how to create a database, create classes, indexes, performs queries and handle queries results.
+`FaunaDB.workspace` contains an iOS Example app that shows the basics of how to use the driver and populates an UITableView with Fauna DB data as well as paginate and filter it. It shows how to create a database, create classes, indexes, performs queries and handle queries results.
+
+You can run the `Example` iOS app in the iPhone simulator by following these steps...
 
-You can run the `Example` iOS app in the iPhone simulator by just cloning this repository, opening the `FaunaDB.workspace` using Xcode and then running `Example` app.
+1. Clone this repository.
+2. Run `carthage update` from the repository root folder.
```

Comment:
Example project uses Carthage because it's a lightweight way to add dependencies. If we were used cocoapods, cocoapods would have created a new workspace and a project just to hold dependencies source files adding overhead to workspace and folders structure. 

On the other hand, carthage is just in charge of building the dependencies and providing the dynamic framework binaries. All example app configuration (where are located the binaries, etc) is done by the developer (i have already configured the example app). So after running `carthage update` command the dynamic library will be available to link and the example app will run by just pressing "build and run" xcode button ( considering we select the proper app project scheme (Example)) .